fips_assessment.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Checkout fips-assessments repo for testing tasks
+- name: git checkout fips-assessment tests
+  ansible.builtin.git:
+    repo: https://github.com/opdev/fips-assessments.git
+    dest: /tmp/fips-assessments/
```

Comment:
Does this ever get cleaned up? We should not leave any artifacts behind.

Reply:
I thought we're supposed to run only with ansible-navigator. Being a container image it gets cleaned up. But you're right if someone runs locally it will remain there.

Reply:
I'll add a clean up task for this.

Reply:
Done!

Reply:
I still use ansible-playbook directly, so... :)